cfme/common/vm_views.py

```diff
@@ -355,9 +355,7 @@ def before_fill(self, values):
                 self.parent.sidebar.decrease_button.click()
                 self._select_template(template_name, provider_name)
             self.continue_button.click()
-            # TODO timing, wait_displayed is timing out and can't get it to stop in is_displayed
-            sleep(3)
-            self.flush_widget_cache()
+            self.plugin.ensure_page_safe()
```

Comment:
```suggestion
            self.browser.plugin.ensure_page_safe()
```

Reply:
Thanks @jan-zmeskal, realized after updating that this is some new GH thing for suggested change - would have clicked 'apply suggestion' otherwise.

Debugging, looks like this may actually be related to the rowspan implementation. Its trying to check table elements before `child_widget_accessed` is executed, so the widget isn't displayed.

Reply:
https://github.com/RedHatQE/widgetastic.core/pull/121